pepper-apis/api-build.sh

```diff
@@ -298,6 +303,12 @@ GIT_SHA_SHORT=${GIT_SHA:0:12}
 shift
 shift
 shift
+# Sneaking in another optional config parameter here
+if grep -q -v "\-\-" <<< "$1"; then
+  APPLICATION=$1
+  shift
+fi
+
```

Comment:
A little hacky but I am hoping we can streamline the script at some point. A lot of the functionality here is no longer relevant.